game_event/tests.py

```diff
@@ -179,3 +198,86 @@ def test_post_game_event(self, client, saved_court, court_ball_game):
         assert GameEvent.objects.count() == count_of_game_events + 1
         assert response.status_code == 302
         assert response.url == '/game-events/'
+
+
+@pytest.mark.django_db
+class TestGameEventViews:
+
+    def test_success_loading_site_game_event(self, client, saved_game_event, player):
+        client.force_login(player.user)
+        game_event_id = saved_game_event.id
+        url = reverse('game_event', args=[game_event_id])
+        response = client.get(url)
+        assert response.status_code == 200
+
+    def test_fail_loading_site_game_event(self, client, player):
+        client.force_login(player.user)
+        response = client.get('/game-events/-1/')
+        assert response.status_code == 404
+
+    def test_success_loading_site_join_game_event(self, client, saved_game_event, player):
+        client.force_login(player.user)
+        game_event_id = saved_game_event.id
+        url = reverse('join_event', args=[game_event_id])
+        response = client.get(url)
+        assert response.status_code == 200
+
+    def test_fail_loading_site_join_game_event(self, client, player):
+        client.force_login(player.user)
+        response = client.get('/game-events/join-event/9999/')
+        assert response.status_code == 404
+
+    def test_success_loading_site_remove_game_event(self, client, saved_game_event, saved_game_event_player):
+        client.force_login(saved_game_event_player.player.user)
+        game_event_id = saved_game_event.id
+        url = reverse('remove_from_event', args=[game_event_id])
+        response = client.get(url)
+        assert response.status_code == 200
+
+    def test_fail_loading_site_remove_game_event_from_player(self, client, player):
+        client.force_login(player.user)
+        response = client.get('/game-events/removee-from-event/-1/')
+        assert response.status_code == 404
+
+    def test_remove_landing_site_game_event(self, client, saved_game_event, saved_game_event_player):
+        client.force_login(saved_game_event_player.player.user)
+        game_event_id = saved_game_event.id
+        url = reverse('remove_from_event', args=[game_event_id])
+        response = client.get(url)
+        assert response.status_code == 200
+        saved_game_event.delete()
+        response = client.get(url)
+        assert response.status_code == 404
```

Comment:
Is this part of the functionality you want to test here? is there some added value for deleting the game event and validating its removal? If not, I suggest removing these lines.

Reply:
that's exactly the the functionality we wanted, @amitCohen2 suggested to add this test.

Reply:
ok. It's not really clear to me what is being tested on this test case.

Reply:
Amit requested to add a test to check whether you can reach the remove_from_event page after the game event has been deleted. should I change the name of the test?

Reply:
ok. I think I understand now. Yes. I think you should rename the test.